docs/eventing/getting-started.md

```diff
@@ -0,0 +1,438 @@
+
+This guide uses the default namespace, so there is no need to create one. 
+
+
+1. Set up the namespace for Knative Eventing. To set up the namespace, add a label to your namespace with this command:
+
+```sh
+kubectl label namespace default knative-eventing-injection=enabled
+```
+
+Labeling a namespace adds a specific organizational structure to the namespace. The `knative-eventing-injection` label triggers Knative to add `Service Accounts`, `Role Bindings`, and a `Broker` to your namespace. Without this label, Knative will not know how to manage your events.
+
+In the next section, you'll learn more about one of the subcompoents added to your namespace, the `Broker`.
+
+### Validating that the `Broker` is running
+
+The `Broker` ensures that every event sent by event producers is sent to the correct event consumers. While you created the `Broker` when you labeled your namespace as ready for eventing, it is important to verify that your `Broker` is working correctly. In this guide, you will use the default broker.
+
+1. Use the following command to verify that the `Broker` is in a healthy state:
+
+```sh
+kubectl -n default get Broker default
+```
+
+This should return the following:
+
+```sh
+NAME      READY   REASON   HOSTNAME                                                           AGE
+default   True             default-Broker.default.svc.cluster.local   1m
+```
+
+When the `Broker` is **READY**, it can begin to manage the events it receives.
+
+2. If the **READY** column reads **False**, wait 2 minutes and repeat Step 1. If the **READY** column still reads **False**, see the [Debugging Guide](./debugging/README.md) to troubleshoot the issue.
+
+Now your `Broker` is ready to manage your events.
+
+### Creating event consumers
+
+These subcomponents receive the events sent by event producers (you'll create those a little later). You'll create two event consumers, `foo-display` and `bar-display`, so that you can see how to selectively send events to different consumers later.
+
+
+1. To create the `foo-display` subcomponent, enter the following command:
+
+```sh
+kubectl -n default apply -f - << END
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: foo-display
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: foo-display
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      containers:
+        - name: event-display
+          # Source code: https://github.com/knative/eventing-sources/blob/release-0.6/cmd/event_display/main.go
+          image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display@sha256:37ace92b63fc516ad4c8331b6b3b2d84e4ab2d8ba898e387c0b6f68f0e3081c4
+
+---
+
+# Service pointing at the previous Deployment. This will be the target for event
+# consumption.
+kind: Service
+apiVersion: v1
+metadata:
+  name: foo-display
+spec:
+  selector:
+    app: foo-display
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
+END
```

Comment:
Why not use a ksvc here instead of a standard deployment?

Reply:
@akashrv , @grantr , @adamharwayne  -- thoughts on this?

Reply:
We totally could use a ksvc instead, but then that would also need to be a prerequisite. I have tended to using K8s svc + deployment to ease debugging, but I don't think this guide is worried about debugging, so it may not be relevant here. 

Please use @Harwayne instead of @adamharwayne.

Reply:
I would expect that since Knative is mainly focused on enabling serving + eventing together, our guide would leverage Knative components when reasonable, especially for a getting started with eventing topic. Perhaps there's another topic we could write about using Knative Eventing with K8S deployments too?

Reply:
Since this guide is focused on eventing, we've decided to keep it simple here and stick with a K8s service (after consulting with @akashrv ).